pkg/tfbridge/info.go

```diff
@@ -85,6 +85,8 @@ type ResourceInfo struct {
 	Docs                *DocInfo               // overrides for finding and mapping TF docs.
 	DeleteBeforeReplace bool                   // if true, Pulumi will delete before creating new replacement resources.
 	Aliases             []AliasInfo            // aliases for this resources, if any.
+	Deprecated          bool                   // if true, the resource will be marked as deprecated
+	Message             string                 // message to use in deprecation warning
```

Comment:
Consider calling this DeprecationMessage.

Reply:
Also - can we just combine these two fields down to just one?

Reply:
Happy to make the change - just pushed it - if we have a `DeprecationMessage` we are now marking something as `Deprecated` no need for the 2